templates/mailhog/0/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  mailhog:
+    image: mailhog/mailhog:v1.0.0
+    environment:
+      MH_UI_WEB_PATH: ${mh_ui_web_path}
+      MH_SMTP_BIND_ADDR: ${mh_smtp_bind_addr}
+    stdin_open: true
+    tty: true
+    user: root
```

Comment:
Is "root" user really necessary?

Reply:
When running the container on SMTP's default port 25, the container must run as root due to permissions issues. Without the root user specified, the container will not run as expected. If you would like, I can open a "to-do" against the MailHog (https://github.com/mailhog/MailHog) project to see if they will accept a change permitting running on port 25 using the standard user. 

The use of root seems to apply in the case of https://github.com/rancher/community-catalog/blob/666a687a3a03b9a6923b796abb192c9ce04dd59d/templates/aqua/1/docker-compose.yml. Hoping we can get this approved and enhance in the future.